engine.py

```diff
@@ -550,12 +547,20 @@ def _emit_log_message_FUTURE(self, handler, record):
         """
 
         # Give a standard format to the message:
-        #     Shotgun DEBUG <basename>: <message>
+        #     Shotgun DEBUG (<timing>) <basename>: <message>
         # or
         #     Shotgun <basename>: <message>
-        msg = "Shotgun %s%s: %s" % ("DEBUG " if record.levelno < logging.INFO else "",
-                                    record.basename,
-                                    record.message)
+        # where "basename" is the leaf part of the logging record name,
+        # for example "tk-multi-shotgunpanel" or "qt_importer".
+        if record.levelno < logging.INFO:
+            current_time_stamp = time.time()
+            msg = "Shotgun DEBUG (%0.3fs) %s: %s" % (current_time_stamp-self._debug_msg_time_stamp,
```

Comment:
@manneohrstrom What about the `record.basename` part of the log line?
Should it be there only for debug messages? Or all the time? Or never?